mithril-aggregator/src/http_server/routes/epoch_routes.rs

```diff
@@ -126,6 +150,95 @@ mod tests {
         .unwrap();
     }
 
+    #[tokio::test]
+    async fn test_epoch_settings_get_ok_with_cardano_transactions_enabled() {
+        let method = Method::GET.as_str();
+        let path = "/epoch-settings";
+        let mut dependency_manager = initialize_dependencies().await;
+        let fixture = MithrilFixtureBuilder::default().with_signers(5).build();
+        let epoch_service = FakeEpochService::from_fixture(Epoch(5), &fixture);
+        dependency_manager.epoch_service = Arc::new(RwLock::new(epoch_service));
+        dependency_manager
+            .signed_entity_config
+            .allowed_discriminants =
+            BTreeSet::from([SignedEntityTypeDiscriminants::CardanoTransactions]);
+        let signing_config = CardanoTransactionsSigningConfig {
+            security_parameter: BlockNumber(70),
+            step: BlockNumber(15),
+        };
+        dependency_manager
+            .config
+            .cardano_transactions_signing_config = signing_config.clone();
+
+        let response = request()
+            .method(method)
+            .path(&format!("/{SERVER_BASE_PATH}{path}"))
+            .reply(&setup_router(Arc::new(dependency_manager)))
+            .await;
+
+        let response_body: EpochSettingsMessage = serde_json::from_slice(response.body()).unwrap();
+
+        assert_eq!(
+            response_body.current_cardano_transactions_signing_config,
+            Some(signing_config)
+        );
+
+        APISpec::verify_conformity(
+            APISpec::get_all_spec_files(),
+            method,
+            path,
+            "application/json",
+            &Null,
+            &response,
+            &StatusCode::OK,
+        )
+        .unwrap();
+    }
+
+    #[tokio::test]
+    async fn test_epoch_settings_get_ok_with_cardano_transactions_not_enabled() {
```

Comment:
Is it useful to have two tests that verify conformity with OpenAPI ? 
Maybe we can check the two cases in in unit test on `epoch_settings` function.

Reply:
Done ✅